src/content/docs/logs/ui-data/query-syntax-logs.mdx

```diff
@@ -18,7 +18,13 @@ Ready to get started? If you haven't already, be sure to [sign up for a New Reli
 
 ## Query structure [#query-structure]
 
-Using the Logs UI, you can search through your log data by entering either simple keywords, such as `new` and `relic`, or phrases such as `new relic agent`, directly into the search field. You can also combine keywords or phrases with operators to form more complex queries.
+Using the Logs UI, you can search through your log data by entering either simple keywords, such as `new` and `relic`, or phrases such as `"new relic agent"`, directly into the search field. 
+
+Plain terms are a "contains" search for the `message` attribute of your logs. For instance, `"New Relic Agent"` is equivalent to the more verbose `message: "*New Relic Agent*"`.  
```

Comment:
I don't think we properly explain how plain terms work in this doc. So adding some notes here.

Reply:
Very helpful. Thanks again!